SharperUniverse.Tests/PersistenceTests.cs

```diff
@@ -198,13 +189,10 @@ public async Task CanLoadPartialState()
 			await fooSystem.RegisterComponentAsync(fooComponent);
 
 			var provider =
-				new LiteDbProvider(new List<ISharperSystem> {fooSystem}, runner)
-				{
-					ConnectionString = $"{Path.GetTempPath()}/SU.db"
-				};
+				new LiteDbProvider(new List<ISharperSystem> {fooSystem}, runner);
 
 
-			var id = provider.Save(new List<BaseSharperComponent>
+			var id = await provider.Save(new List<BaseSharperComponent>
 			{
```

Comment:
Inconsistency with the instantiation on line 159 of this same file. Please keep it uniform.